extensions/amp-story/amp-story-ads.md

```diff
@@ -0,0 +1,176 @@
+# Advertising support in AMP Stories 
+
+## AMP Story Ad Principles
+
+The current ad ecosystem of banners and boxes doesn't integrate well with the rest of the content around it. Ads feel out of place, are slow and interruptive and don't feel native to the rest of the story experience.
+
+We'd like to establish the following principles for Story Ads:
+
+* Visual First:  Inviting, bold, context driven invitation state
+* Native: The ad page has the same dimensions as an organic story page. 
+* Same Interaction Model: User can continue to the next screen just like they would with an organic story page
+* Fast: The ad will never appear to a user in a half-loaded state. 
+
+As a result, one key difference from regular web pages is that the AMP Stories runtime determines the right placement of the ad page amidst the AMP Story. These mechanics are explained as part of [amp-story-auto-ads](./amp-story-auto-ads.md)
+
+
+## Ad Formats
+There are two type of ad formats supported as part of AMP Stories:
+
+* Single Page Ad : Where the ad appears as a single page inside of an AMP story 
+* Sponsored Story Ad: Where the ad is a stand-alone multi-page story.
+
+### Single Page Ad 
+
+
+In-line with the principles, a single page ad appears in between organic story content as a full page. 
+Single Page Ads have a predefined set of call to action buttons and those call to action buttons take a web landing page URL where the user is navigated to, on click. 
+![alt text](https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/story-page-ad.png "Story Page Ad")
+
+
+### Sponsored Story Ad 
+When publishing sponsored story ads, use the `<amp-story ad>` tag instead of `<amp-story>`, so platforms can surface these stories appropriately. Since a Sponsored Story Ad is just a URL on the web, you can also send user traffic to a sponsored story ad from the CTA URL button of a single page ad. 
+![alt text](https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/sponsored-story-ad.png "Sponsored Story Ad")
+
+## Consistent UX
+
+The ad label and the call to action (CTA) buttons on the ads must be consistent across all publishers and ad networks. Therefore, the Stories runtime takes responsibility of rendering those in a consistent manner. 
+
+![alt text](https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/consistent-ux.png "Consistent Ad UX")
+
+
+## Publisher placed ads 
+This is an option for publishers who would like to place single page ads amidst AMP Story content they produce. This is meant as a stop-gap until more robust ad server support is available. 
+
+This is done using the mechanism in [Custom ad](../../ads/custom.md) extension. The ads are rendered with inlined templates in the story document
+and the data for the templates is fetched remotely.
```

Comment:
Was talking to @lannka today about this: I had some confusion understanding that they should have one endpoint that returns different ad payloads on each request. Maybe we could make this more clear somehow?